### PR TITLE
[[ EmscriptenWindows ]] Fix incomplete raisewindow implementation

### PR DESCRIPTION
**Pull request by livecodeian**
depends on PR https://github.com/livecode/livecode/pull/6097

### CODE REVIEW

File: engine/src/w32stack.cpp [hunk 1]
```diff
@@ -474,6 +474,11 @@ void MCStack::sethints()
 		SetWindowLongA((HWND)window->handle.window, 0, (LONG)sptr->getw()->handle.window);
 }
 
+bool MCStack::view_platform_dirtyviewonresize()
```
Comment (livecodeali) [inline engine/src/w32stack.cpp]: This is still missing a `const`

**Comment by runrevmark** [comment]
@livecode-vulcan review ok dedef35

**Comment by livecode-vulcan** [comment]
💙 review by @runrevmark ok dedef35463e569260f935265ad8786ddd7c1c539

**Comment by livecode-vulcan** [comment]
😎 test success dedef35463e569260f935265ad8786ddd7c1c539

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/1082)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/1098)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/1072)
* try-community-universal-ios-iphoneos11.2: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos11.2/builds/25)
* try-community-universal-ios-iphonesimulator11.2: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator11.2/builds/25)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/1077)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/1099)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/1082)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/1098)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/1081)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/1091)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/1114)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/1077)

**Pull request merged**